### PR TITLE
Added evigene/tr2aacds

### LINKED ISSUE #4737: new module: evigene/tr2aacds
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by GallVp**
## PR checklist

Closes #4737 

- [x] Added evigene/tr2aacds
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by GallVp** [comment]
For visibility, @avani-bhojwani

File: modules/nf-core/evigene/tr2aacds/main.nf [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    \$EVIGENEHOME/scripts/prot/tr2aacds.pl \\
+        $args \\
+        -NCPU=$task.cpus \\
+        -MAXMEM=$max_memory \\
+        -cdnaseq $fasta
+
+    find \\
+        dropset \\
+        -type f \\
+        -exec sh -c 'mv "\$1" "\$(echo \$1 | sed s/$simple_name/$prefix/)"' sh {} \\;
```
Comment (SPPearce) [inline modules/nf-core/evigene/tr2aacds/main.nf:51]: What happens if simple_name == prefix?
I think that conda fails if you try and mv the file to itself.

File: modules/nf-core/evigene/tr2aacds/meta.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "evigene_tr2aacds"
+description: Uses evigene/scripts/prot/tr2aacds.pl to filter a transcript assembly
+keywords:
+  - sort
+  - example
+  - genomics
```
Comment (SPPearce) [inline modules/nf-core/evigene/tr2aacds/meta.yml:8]: Are these the default keywords?

File: modules/nf-core/evigene/tr2aacds/main.nf [hunk 3]
```diff
@@ -0,0 +1,104 @@
+process EVIGENE_TR2AACDS {
+    tag "$meta.id"
+    label 'process_high'
```
Comment (SPPearce) [inline modules/nf-core/evigene/tr2aacds/main.nf:3]: Does this need process_high?

File: modules/nf-core/evigene/tr2aacds/main.nf (hunk 1 above)
Comment (GallVp) [reply modules/nf-core/evigene/tr2aacds/main.nf]: Good point @SPPearce 

I have added a conditional to cater to such cases.

File: modules/nf-core/evigene/tr2aacds/meta.yml (hunk 2 above)
Comment (GallVp) [reply modules/nf-core/evigene/tr2aacds/meta.yml]: Yes, good spotting. I have updated the keywords now. Thanks!

File: modules/nf-core/evigene/tr2aacds/main.nf (hunk 3 above)
Comment (GallVp) [reply modules/nf-core/evigene/tr2aacds/main.nf]: No sure. The help section does have examples with 8 and 30 CPUs. I have now used `process_medium`. @avani-bhojwani can you please experiment with your data when possible and update the label if needed? Thanks!

Comment (avani-bhojwani) [reply modules/nf-core/evigene/tr2aacds/main.nf]: I've been using process_medium and it seems to work well

**Review by SPPearce — APPROVED** [review]
Just to clarify, if process high is appropriate that is fine, I just always ask the question as it requests a lot of resources.

**Comment by GallVp** [comment]
It is good to check. Thanks for your help.

**Pull request merged**